velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -372,4 +372,18 @@ struct AddMonthsFunction {
     result = daysSinceEpoch;
   }
 };
+
+template <typename T>
+struct MonthFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE int32_t getMonth(const std::tm& time) {
```

Comment:
It looks we can just use a function in the below, then no need to define a new one.
https://github.com/facebookincubator/velox/blob/4d7f31c9b21819dc8ed1e9d16a38b432b6e67b28/velox/functions/lib/TimeUtils.h#L76

Reply:
@PHILO-HE updated.